app/src/main/java/com/treasurehunt/data/remote/FirebaseRepository.kt

```diff
@@ -0,0 +1,17 @@
+package com.treasurehunt.data.remote
+
+import android.util.Log
+import com.treasurehunt.data.remote.model.UserDTO
+
+class FirebaseRepository(private val firebaseService: FirebaseService) {
```

Comment:
강사님이 최근에 피드백 주신 내용이기도 한데, 레포지토리 이름에 서비스 이름 (implementation detail)을 넣는 건 비추로 알고 있습니다
공식문서에도 이와 같은 내용 있으니 참고해주세요
user, log, place 각각에 해당하는 레포지토리가 따로 있어야 할것 같고, 추후에 로컬Db 브랜치와 병합되면, 각각이 또 local, remote로 나뉘거나 아니면 통합해서 사용하면 될 거 같습니다

In this guide, repository classes are named after the data that they're responsible for. The convention is as follows:

type of data + Repository.

For example: NewsRepository, MoviesRepository, or PaymentsRepository.

Don't name the data source based on an implementation detail—for example, UserSharedPreferencesDataSource—because repositories that use that data source shouldn't know how the data is saved. 

https://developer.android.com/topic/architecture/data-layer

Reply:
반영했습니다!